<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoneyTransfer</class>
 <widget class="QDialog" name="MoneyTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Money Transfer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="from_list">
       <property name="currentText">
        <string/>
       </property>
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>---&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="to_list">
       <property name="duplicatesEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDateEdit" name="date">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Amount: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="value">
       <property name="maximum">
        <double>10000000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Description:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="description"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="execute_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Execute Transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MoneyTransfer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>execute_button</sender>
   <signal>clicked()</signal>
   <receiver>MoneyTransfer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>from_list</sender>
   <signal>activated(int)</signal>
   <receiver>MoneyTransfer</receiver>
   <slot>fromActivated(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_list</sender>
   <signal>activated(int)</signal>
   <receiver>MoneyTransfer</receiver>
   <slot>toActivated(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fromActivated(int)</slot>
  <slot>toActivated(int)</slot>
 </slots>
</ui>
